shelter/remoteattestation/ca_cert.go

```diff
@@ -0,0 +1,33 @@
+package remoteattestation
+
+var caCert = []byte(`-----BEGIN CERTIFICATE-----
```

Comment:
What is the purpose of this variable?

Reply:
Actually, it is unnecessary and no use in current stage. In the start of project, i think it will be used to verify x509 certification.  I will remove this file.